ui/.env.production

```diff
@@ -2,3 +2,22 @@ REACT_APP_URL=http://localhost:8080
 REACT_APP_API=http://localhost:8080/api
 REACT_APP_AUTH0_DOMAIN=jobsgowhere.au.auth0.com
 REACT_APP_AUTH0_CLIENT_ID=rc3Waw3xebsUoG2uE7I2Zb1T5jgSfrTF
+REACT_APP_FIREBASE_API_KEY=AIzaSyBoW5rHHEmI9TqvSWFbYaprj4HV3RhtlIw
+REACT_APP_FIREBASE_AUTH_DOMAIN=jobsgowhere-dev.firebaseapp.com
+REACT_APP_FIREBASE_DATABASE_URL=https://jobsgowhere-dev.firebaseio.com
+REACT_APP_FIREBASE_PROJECT_ID=jobsgowhere-dev
+REACT_APP_FIREBASE_STORAGE_BUCKET=jobsgowhere-dev.appspot.com
+REACT_APP_FIREBASE_SENDER_ID=243712014749
+REACT_APP_FIREBASE_APP_ID=1:243712014749:web:cc46186279fa928694d774
+REACT_APP_FIREBASE_MEASUREMENT_ID=G-V9H05XKFW0
+REACT_APP_GTM_ID=GTM-MDB42H4
+POSTGRES_CONNECTION_STRING=postgresql://postgres:postgres@localhost/jobsgowhere?sslmode=disabled
+AUTH0_DOMAIN=jobsgowhere.au.auth0.com
+AUTH0_CLIENT_ID=rc3Waw3xebsUoG2uE7I2Zb1T5jgSfrTF
+AUTH0_AUDIENCE=jobsgowhere
+AUTH0_CALLBACK=http://localhost:8080
+DATABASE_URL=postgres://postgres@localhost:5432/jobsgowhere?sslmode=disable
+LINKEDIN_CLIENT_ID=
+LINKEDIN_CLIENT_SECRET=
+LINKEDIN_CALLBACK_HOST=http://localhost:8080
+SENDGRID_API_KEY=
```

Comment:
comments are same as `.env.development` ... looks like these credentials are duplicate and not needed.

Reply:
Removed redundant variables.